mass_mailing_sending_queue/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_mail_mass_mailing_sending_admin,mail_mass_mailing_sending group_system,model_mail_mass_mailing_sending,base.group_system,1,1,1,1
```

Comment:
Is there a reason for admins to be the only ones that can see this? Shouldn't this be `base.group_user`?

0. Log in runbot as demo user.
0. Go to Marketing > Mass mailing.
0. Click on "First Newsletter".

```
AccessError

Sorry, you are not allowed to access this document. Only users with the following access level are currently allowed to do that:
- Administration/Settings

(Document model: mail.mass_mailing.sending) 
```